src/shared/layouts/MainLayout.tsx

```diff
@@ -86,7 +96,99 @@ const MainLayout: React.FunctionComponent<MainLayoutProps> = ({
+          routes={[
+            {
+              path: '/',
+              component: (path, goTo) => {
+                return (
+                  <div className="page -home">
+                    <p>Projects are where you group and categorize data</p>
+                    <Button icon="project" onClick={() => goTo('/selectOrg')}>
+                      Select a Project
+                    </Button>
+                    <Divider />
+                    <RecentlyVisited visitProject={goToProject} />
+                  </div>
+                );
+              },
+            },
+            {
+              path: '/selectOrg',
+              component: (path, goTo) => {
+                return (
+                  <div className="page -select-org">
+                    <h4 className="title">
+                      <Button
+                        size="small"
+                        onClick={() => goTo('/')}
+                        icon="arrow-left"
+                      ></Button>{' '}
+                      Select an Organziation
+                    </h4>
+                    <OrgList>
+                      {({ items }: { items: OrgResponseCommon[] }) =>
+                        items.map(i => (
+                          <ListItem
+                            key={i['@id']}
+                            onClick={() => goTo(`/selectProject/${i._label}`)}
+                          >
+                            <OrgItem {...i} />
+                          </ListItem>
+                        ))
+                      }
+                    </OrgList>
+                  </div>
+                );
+              },
+            },
+            {
+              path: '/selectProject/:orgLabel',
+              component: (path, goTo, { orgLabel }) => {
+                return (
+                  <div className="page -select-project">
+                    <h4 className="title">
+                      <Button
+                        size="small"
+                        onClick={() => goTo('/')}
+                        icon="arrow-left"
+                      ></Button>{' '}
+                      Select a Project
+                    </h4>
+                    <ProjectList orgLabel={orgLabel}>
+                      {({ items }: { items: ProjectResponseCommon[] }) =>
+                        items.map(i => (
+                          <ListItem
+                            key={i['@id']}
+                            onClick={() =>
+                              goToProject(i._organizationLabel, i._label)
+                            }
+                          >
+                            <ProjectItem {...i} />
+                          </ListItem>
+                        ))
+                      }
+                    </ProjectList>
+                  </div>
+                );
+              },
+            },
+          ]}
+          defaultVisibility={false}
+          render={(visible, toggleVisibility) => {
```

Comment:
I had to come down here in order to understand this prop 😝 
I would just remove it. Why not simply have a `visible: boolean` prop instead?
Like these guys: https://ant.design/components/drawer/

Reply:
because I wanted to move the toggle logic to the component. You always need a toggle and it's a pain to include toggle state handling everywhere you use such a thing, no?

Reply:
Well, I don't find this:

```tsx
const [visible, setVisible] = useState(false);
return (
  <Button
    icon="project"
    size="small"
    onClick={() => setVisible(!visible)}
  >
    Projects
  </Button>
 );
```
to be more painful than:

```tsx
render={(visible, toggleVisibility) => {
            return (
              <Button
                icon="project"
                size="small"
                onClick={() => toggleVisibility()}
              >
                NavDrawer
              </Button>
            );
          }}
```

Also, what if I don't need a toggle component and I just want it to be visible at all time?